runtime/doc/api-ui-events.txt

```diff
@@ -276,6 +276,10 @@ the editor.
 	to an internal buffer, this is the time to display the redrawn parts
 	to the user.
 
+["tty_write", content] ~
```

Comment:
We have `nvim_ui_term_event` which refers to the host terminal as `term`. We probably should have named that `nvim_ui_tty_event` I guess. Or we could name this `term_write`, but we have been pretty consistent about referring saying "the terminal" only when referring to `:terminal`, and "the TUI" when referring to the host-terminal. Though `tty` is a somewhat helpful clarification for low-level contexts like this and `stdout_tty`.

Objections to naming this `ui_write` ?

Reply:
I considered `ui_write` but felt like this API is meaningless for UIs that don't have a TTY connected to their stdout.

I'm not opposed to it. Just not sure what the use case is.

>Or should nvim_ui_term_event be renamed to nvim_ui_tty_event ?

I don't feel strongly about this either way. One point in favor of `term_event` is consistency with `v:termresponse`, `TermResponse`, `TermRequest`, etc.